From d3c7fdf3059edb55336559c13768fe22fd369b13 Mon Sep 17 00:00:00 2001
From: Andrew Jeffery <andrew@aj.id.au>
Date: Fri, 21 Feb 2020 14:35:39 +1030
Subject: [PATCH 5/6] console-client: Improve failure-to-connect warning

Print the abstract socket path to which we failed to connect.

Signed-off-by: Andrew Jeffery <andrew@aj.id.au>
Change-Id: I62b3656d5aeda655dcb89e53beccb02f0046661c
---
 console-client.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/console-client.c b/console-client.c
index 1d0a48934f4b..1f8517611019 100644
--- a/console-client.c
+++ b/console-client.c
@@ -210,6 +210,7 @@ static int client_tty_init(struct console_client *client)
 static int client_init(struct console_client *client, const char *socket_id)
 {
 	struct sockaddr_un addr;
+	socket_path_t path;
 	ssize_t len;
 	int rc;
 
@@ -235,7 +236,8 @@ static int client_init(struct console_client *client, const char *socket_id)
 	if (!rc)
 		return 0;
 
-	warn("Can't connect to console server");
+	console_socket_path_readable(&addr, len, path);
+	warn("Can't connect to console server '@%s'", path);
 cleanup:
 	close(client->console_sd);
 	return -1;
-- 
2.20.1

